src/sentry/snuba/discover.py

```diff
@@ -1004,3 +1004,26 @@ def get_facets(query, params, limit=10, referrer=None):
         )
 
     return results
+
+
+def get_related_issues(params, lookup_keys):
```

Comment:
Should we use issue search for this instead of building more methods? We could create an issue search query based on the transaction name and `params`.